### PR TITLE
created and styled modals for profile page

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: js/components/NoteList/AddNote/AddNote.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ export default class AddNote extends Component {
     };
   }
   render() {
-    const { open, close } = this.props;
+    const { open, close, showX } = this.props;
```
Comment (siddharthparmar7) [inline js/components/NoteList/AddNote/AddNote.js]: You do not need open and close both props. Just have one.

File: js/components/NoteList/AddNote/AddNote.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ export default class AddNote extends Component {
...
     return (
       <View style={styles.modalContainer}>
         <Modal
```
Comment (siddharthparmar7) [inline js/components/NoteList/AddNote/AddNote.js]: your Modal is your wrapper, no need to add a View to wrap everything.

File: js/components/TagModal/TagModal.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import { View, Text, TouchableOpacity, Modal } from 'react-native';
+import styles from './styles';
+import PropTypes from 'prop-types';
+
+const TagModal = ({ open, close, title, description, backgroundColor }) => {
```
Comment (siddharthparmar7) [inline js/components/TagModal/TagModal.js]: you do not need open and close both as props. Just have one bool to help you. If close is a method, name it something else like closeModal.

File: js/components/TagModal/styles.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import { StyleSheet, Dimensions } from 'react-native';
+import { colors, typography } from '../../config/styles';
+
+const { width, height } = Dimensions.get('window');
```
Comment (siddharthparmar7) [inline js/components/TagModal/styles.js]: width is not being used. Please remove it.

File: js/components/TagModal/TagModal.js (hunk 3 above)
Comment (siddharthparmar7) [inline js/components/TagModal/TagModal.js]: the background color is not being used. Please remove it.

File: js/components/TagModal/TagModal.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
```
Comment (siddharthparmar7) [inline js/components/TagModal/TagModal.js]: Component is not being used. Please remove that import.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
You should not have two components inside a component dir. Please move Note and AddNote out of NoteList.

Please make the requested changes. Thank you!

File: js/components/TagModal/TagModal.js (hunk 3 above)
Comment (ghost) [reply js/components/TagModal/TagModal.js]: It will be used when implemented in the page that's why we kept it .

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
your prop `close` should be named `closeModal` for better clarity on what close is?

Also, as I asked earlier please move your nested component dirs to component dir. There should be no nesting in components dir. 

Thank you.

**Review by siddharthparmar7 — APPROVED** [review]
Thank you! Nice work!

**Pull request merged**